action.yml

```diff
@@ -8,6 +8,9 @@ inputs:
   liv-doc-regime:
     description: 'Enable or disable the LivDoc regime.'
     required: true
+  output-formats:
+    description: 'Comma-separated list of wanted output formats.'
+    required: true
```

Comment:
When required, then validation is missing. I guess calling sys.exit(1) is not the correct solution.
Return None seem to be better options.
Keep in mind that will exist is in ActionInputs validation logic when a problem found.

Reply:
I did move the validation into the ActionInput validation logic, which is called in the begging of the generation process. You can see the logic and suggest any other forms of validation. Commit: [0903d56](https://github.com/AbsaOSS/living-doc-generator/pull/83/commits/0903d56280023e868d2272718108495727210919).